packages/react-scripts/template/README.md

```diff
@@ -23,7 +23,7 @@ You can find the most recent version of this guide [here](https://github.com/fac
 - [Adding a Stylesheet](#adding-a-stylesheet)
 - [Post-Processing CSS](#post-processing-css)
 - [Adding a CSS Preprocessor (Sass, Less etc.)](#adding-a-css-preprocessor-sass-less-etc)
-- [Adding Images and Fonts](#adding-images-and-fonts)
+- [Adding Images, Fonts, and Files](#adding-images-fonts-and-files)
```

Comment:
Sorry I wasn't clear. I meant that we also have these links in README.md in the root. I updated it: https://github.com/facebookincubator/create-react-app/commit/7dd124c8dd5a0a6af83de51d345e5647e751f593.